docs/user-manual/en/mfa.md

```diff
@@ -0,0 +1,44 @@
+# Multi Factor Authentication
+
+Kapua provides a Multi Factor Authentication feature (a.k.a. MFA), which allows a user to authenticate to a system only when he presents the 
+password and another valid factor. A user can enable the MFA through the User menu in the Console.
+
+![MFA User View](images/mfa_user_view.png)
+
+The user can also access the MFA configuration dialog through the upper right menu. Note that only the user itself can enable the MFA.
+
+![MFA Enable](images/mfa_enable.png)
+
+MFA uses a secret to produce a one-time code to authenticate. The secret will be displayed in the form of a QR code during the MFA activation. 
+This QR code can be viewed just once during MFA activation, and only by the user itself. To collect the QR code, use an authenticator app 
+(e.g. the Google Authenticator App) installed on a smartphone. This app will be used to produce one-time authentication code.
+
+Scratch codes are also produced, in order to pass the multi factor authentication in case the authenticator app is temporary unavailable. 
+Similarly, to the QR code, also scratch codes can be viewed just once during MFA activation, and they can be viewed only by the user itself. 
+Users have a limited number of scratch codes (the maximum number of scratch codes can be set through a dedicated environment variable, see the Web 
+Console container properties section for more information). Furthermore, each scratch code is invalidated after being used once.
+
+![MFA QR Code](images/mfa_qr_code.png)
+
+The admin is able to see if the user have the MFA enabled or not. Moreover, even if only the user is allowed to enable MFA, the admin is allowed to disable 
+it for each one of the users in the account.
+
+When the MFA is enabled for a given user, such user have to type the one-time authentication code provided by the authenticator app as part of the login 
```

Comment:
Instead "...such user have to type..." -> "...such user has to type..."

Reply:
My fault, done.